public/editor/stylesheets/editor.less

```diff
@@ -850,13 +850,14 @@ body {
 }
 
 .transition {
-    -webkit-transition: @transitionTime;
-    -moz-transition: @transitionTime;
-    -ms-transition: @transitionTime;
-    -o-transition: @transitionTime;
-    transition: @transitionTime;
+    -webkit-transition: @transitionTime ease-out;
+    -moz-transition: @transitionTime ease-out;
+    -ms-transition: @transitionTime ease-out;
+    -o-transition: @transitionTime ease-out;
+    transition: @transitionTime ease-out;
     -webkit-backface-visibility: hidden;
     backface-visibility: hidden;
+	transition-timing-function: ease-out;
```

Comment:
@bkim49 I think you're using tabs vs. spaces. We use 2-space indents throughout the code so would you be able to convert this to 4 spaces?